dino_appdata.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/main/data/im.dino.Dino.appdata.xml b/main/data/im.dino.Dino.appdata.xml
```

Comment:
Has it been submitted upstream yet?

Reply:
https://github.com/dino/dino/pull/1237 i did just now